.gitignore

```diff
@@ -2,4 +2,5 @@
 webdriver-report
 secrets.json
 /*/**/__pycache__/*
+.python-version
```

Comment:
I had set my python version to 3.10 which added the file `.python-version`. This file does not need to be tracked.